index.html

```diff
@@ -0,0 +1,260 @@
+                        <p class="programdescription">E-commerce poses challenges such as security risks, competition, customer trust, and the need for effective digital marketing strategies to stand out in a crowded online marketplace.</p>
+                    </div>
+                    <div class="program">
+                        <img class="programimage" src="./images/program_icon_05 1.png" alt="program feature icon"/>
+                        <h2 class="programheader">Simulation</h2>
+                        <p class="programdescription">business-to-business (B2B) platforms, and digital service providers. These platforms provide an avenue for businesses to reach a global customer base and enable consumers to access a vast array of products and services from around the world.</p>
+                    </div>
+                </div>
+                <a class="LinkProgram" href="#">SEE THE HOLE PROGRAM</a>
+            </div>
+        </section>
+        <section class="SectionSpeakers" id="SectionSpeakers">
+            <div class="ContainerSectionSpeakers">
+                <div class="SectionSpeakT">
+                    Featured Speakers
+                </div>
+                <div class="RedLine"></div>
+                <div class="SpeakersCont" id="speakersContainer"></div>
+                <div class="SeeMoreButon">
+                    <a href="#" class="SeeMoreBtn BtnR" id="seeMoreBtn">
+                        See More
+                    </a>
+                </div>
+            </div>
+        </section>
+        <section class="main1" id="main1">
+            <div class="firsttemplate1">
+                <div class="info1">
+                    <div class="title11">
+                        <h2 class="title2">The E-Commerce </br>
+                            Pioneers Max 2023
+                        </h2>
+                    </div>
+                    <div class="squareInfo1">
+                        <h4 class="resumetitle1">
+                            The Global Summit brings together the community
+                            of experts, academics, and activists who comprise 
+                            the Creative Commons affiliate network in a 
+                            different country every two years. This year, 
+                            we hope to expand our invitation list including 
+                            organizations and individuals.
+                        </h4>
+                    </div>
+                    <div class="dateTitle1" id="dateTitle1">
+                        Please contact us per Email for nay further questions about CC Global Summit 2015!
+                    </div>
+                    <div class="siteDir1">ccsummit2015@anaxtasia.com</div>
+                    <div class="ContainerSectionSpeakers">
+                        <div class="SectionSpeakT1">
+                            Creative Commons Global Summit 2015 Logo
+                        </div>
```

Comment:
- [ ] Kindly ensure you also style your About page to match the given design. The major issue is the font size of your headings. Kindly reduce the font size and center your content for the About page as is in the given design

Your design
![image](https://github.com/jovasal/Capstone/assets/81932478/2e183a9f-2687-4a99-9b76-af8b29442715)

Given design
![image](https://github.com/jovasal/Capstone/assets/81932478/003838a7-7dde-463f-a981-9f05d495a0aa)

Reply:
Hi @Munsa1 I changed this color under recomendación mentioned before. Whereas I changed again under your coments.

![image](https://github.com/jovasal/Capstone/assets/35693330/ec6b04cb-481c-4479-96f3-c79a04900920)

Reply:
Okay here I meant the font size of your title was too big and therefore was distorting the design, I was suggesting that you significantly reduce the font size

Reply:
Also, I meant to suggest that you could align your content to the middle as is in the given design